doc/enterprise-shield/p_enterprise_shield_config.md

```diff
@@ -0,0 +1,418 @@
+    
+         ``` xml:
+         <socks.transport>           
+           wsn+ssl://${dmz.backplane.hostname}:1080/shield         
+         </socks.transport>     
+         ```
+
+        In most cases, using SOCKS is sufficient to establish reverse connections for Enterprise Shield™. However, in some environments, intermediaries like firewalls between the internal Gateway and the DMZ Gateway can interfere with idle connections. Therefore, the best practice is to add a WebSocket transport under the SOCKS layer because you can configure the WebSocket transport to keep idle connections alive. In fact, the next step shows you how to configure this by setting the `ws.inactivity.timeout` element.
+
+        See [socks.transport](../admin-reference/r_configure_gateway_service.md#protocoltransport) for more information about using the scheme `wsn+ssl://` for the reverse connection between the internal Gateway and the DMZ Gateway.
+
+    -   Configure [ws.inactivity.timeout](../admin-reference/r_configure_gateway_service.md#wsinactivitytimeout) to detect network failures between the DMZ and internal Gateway:
+
+       `         <ws.inactivity.timeout>60 seconds</ws.inactivity.timeout>  `
+       
+      Our example specifies that the network connection can be inactive for a maximum of 60 seconds. The internal Gateway drops the connection and retries if communication with the DMZ Gateway is not possible for a period of 60 seconds.
+
+The App1 service is set up on the internal Gateway. Now let’s set up the App2 service.
+
+### Set up the App2 service on the internal Gateway
+
+The service configuration for the App2 service is exactly the same as for App1, except you need to provide a unique name for this service in the `name` and `accept` elements. The following example shows a complete service configuration for App2:
+
+``` xml:
+<service>
+
+  <name>Enterprise App2</name>
+  <accept>wss://gateway.example.com:8443/app2</accept>
+  <connect>tcp://127.0.0.1:3102</connect>
+
+  <type>proxy</type>
+
+  <accept-options>
+    <http.transport>socks://${dmz.backplane.hostname}:1080</http.transport>
+    <socks.mode>reverse</socks.mode>
+    <socks.retry.maximum.interval>1 second</socks.retry.maximum.interval>
+    <socks.transport>
+      wsn+ssl://${dmz.backplane.hostname}:1080/shield
+    </socks.transport>  
+    <ws.inactivity.timeout>60 seconds</ws.inactivity.timeout>
+  </accept-options>
+
+  <cross-site-constraint>
+    <allow-origin>*</allow-origin>
+  </cross-site-constraint>
+
+</service>
+```
+
+As you can see, the accept-options are exactly the same for both the App1 and App2 services.
+
+You've completed configuring the internal Gateway! Now let’s configure the DMZ Gateway.
+
+Step 5: Configure the DMZ Gateway
+---------------------------------------------------------
+
+Perform the following steps to set up the App1 and App2 services in the DMZ Gateway.
+
+### Set up the App1 service on the DMZ Gateway
```

Comment:
Thanks for the reminder (I really do appreciate it).  I stepped through your comments and made sure the appliable comments were applied to the DMZ Gateway steps.